docs/audiences.md

```diff
@@ -14,8 +14,17 @@ const LOOK_BACK_IN_SECS = 60 * 24 * 7;  // 1 week in seconds
 const OCCURRENCES = 2;
 
 export const exampleAudience: AudienceDefinition = {
+  // A map of account ids that the audience definition can run for
+  // If this is not defined then the audience definition will run for all accounts
+  accountId?: {
+    // Set the state for each account to 'live', 'paused' or 'deleted'
+    someAccount: 'live',
+    someOtherAccount: 'paused',
```

Comment:
Open source edgekit does not need to know about accountIds as far as I am aware?